src/components/Main.vue

```diff
@@ -73,10 +79,15 @@ export default {
             rendered: false,
             sidebarStatus: false,
             widthVal: '100%',
-            floatVal: 'none'
+            floatVal: 'none',
+            highlight: "",
```

Comment:
I'm a bit confused about what the type of `highlight` should be. Is it a String or an Object?

Set the default value appropriately, and maybe change the attribute name.